blueman/plugins/applet/Menu.py

```diff
@@ -4,18 +4,94 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-from blueman.Functions import *
+import dbus
```

Comment:
I checked with ``gdbus monitor -e -d org.blueman.Applet`` and I do see it.

```
The name org.blueman.Applet is owned by :1.885
/: org.blueman.Applet.MenuChanged ([{'tooltip': <'Laat netwerkverkeergebruik zien'>, 'markup': <false>, 'sensitive': <true>, 'text': <'Netwerk _gebruik'>, 'icon_name': <'network-wireless'>}],)
```

very confused why it does not show up in ``on_signal``.